testing/ostest/suspend.c

```diff
@@ -22,14 +22,18 @@
  * Included Files
  ****************************************************************************/
 
-#include <sys/types.h>
+#include <nuttx/config.h>
+
+#include <errno.h>
+#include <pthread.h>
+#include <sched.h>
+#include <semaphore.h>
+#include <signal.h>
 #include <stdbool.h>
 #include <stdio.h>
 #include <unistd.h>
-#include <semaphore.h>
-#include <signal.h>
-#include <sched.h>
-#include <errno.h>
+#include <sys/types.h>
```

Comment:
Do we really need this include? I think sys/types.h is included in most of the standard headers above. What specific type used here needs this explicit include?
And same question in other similar places

Reply:
`sys/types.h` is most probably required due to `size_t`, but I agree that there might be some cases where it is needlessly being included.

In this current endeavor I am just trying to make #8744 pass the CI by including the missing headers. Later we could use a tool-assisted method (e.g. [IWYU](https://include-what-you-use.org/) to remove unused headers.